includes/class-newspack-popups-model.php

```diff
@@ -1222,7 +1250,26 @@ class="<?php echo esc_attr( implode( ' ', $classes ) ); ?>"
 			</div>
 			<?php if ( ! $no_overlay_background ) : ?>
 				<div style="opacity: <?php echo floatval( $overlay_opacity ); ?>;background-color:<?php echo esc_attr( $overlay_color ); ?>;" class="newspack-lightbox-shim"></div>
-				</form>
+			<?php endif; ?>
+			<?php if ( 'form_submission' === $dismiss_action ) : ?>
+				<script data-amp-plus-allowed type="text/javascript">
+					var s = document.querySelector('form[data-submit-success]');
```

Comment:
This selector will select the first matching form on the page – if there are multiple prompts with a form, the listener might be attached to a different form.